uzdevums-2/LoanManagementApi/LoanManagementApi/Startup.cs

```diff
@@ -25,6 +23,10 @@ public Startup(IConfiguration configuration)
         public void ConfigureServices(IServiceCollection services)
         {
             services.AddControllers();
+            services.AddDbContext<ManagementDBContext>(opt => opt.UseSqlServer(Configuration.GetConnectionString("ConnectionString")));
+            services.AddScoped<IUserRepository, UserRepository>();
```

Comment:
prieks, ka di lieto servisu interfeisus + to implementācijas, nevis raw implementācijas